contracts/ComptrollerStorage.sol

```diff
@@ -94,4 +72,36 @@ contract ComptrollerV1Storage {
 
     /// @notice True if a certain action is paused on a certain market
     mapping(address => mapping(Action => bool)) internal _actionPaused;
+
+    struct LiquidationOrder {
+        VToken vTokenCollateral;
+        VToken vTokenBorrowed;
+        uint256 repayAmount;
+    }
+
+    struct AccountLiquiditySnapshot {
+        uint256 totalCollateral;
+        uint256 weightedCollateral;
+        uint256 borrows;
+        uint256 effects;
+        uint256 liquidity;
+        uint256 shortfall;
+    }
+
+    struct RewardSpeeds {
+        address rewardToken;
+        uint256 supplySpeed;
+        uint256 borrowSpeed;
+    }
```

Comment:
Let's move these to the beginning of the contract before state variable declarations
https://docs.soliditylang.org/en/v0.8.17/style-guide.html#order-of-layout

Reply:
Fixed , can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/151/commits/ffd493f85fd85b0e1ede68a5b5b8ca0e298698e4).